model/internal/data/protogen/metrics/v1/metrics.pb.go

```diff
@@ -893,21 +940,14 @@ func (m *Summary) GetDataPoints() []*SummaryDataPoint {
 // Deprecated: Do not use.
 type IntDataPoint struct {
 	// The set of labels that uniquely identify this timeseries.
-	Labels []v11.StringKeyValue `protobuf:"bytes,1,rep,name=labels,proto3" json:"labels"`
-	// start_time_unix_nano is the last time when the aggregation value was reset
-	// to "zero". For some metric types this is ignored, see data types for more
-	// details.
-	//
-	// The aggregation value is over the time interval (start_time_unix_nano,
-	// time_unix_nano].
+	Labels []*v11.StringKeyValue `protobuf:"bytes,1,rep,name=labels,proto3" json:"labels,omitempty"`
```

Comment:
Why did this need to change? Can we keep `Labels []v11.StringKeyValue` and avoid the need to have `stringKeyValuePtrArrayToStringKeyValueArray` func?

Reply:
🤔 that's a good question, this is all generated via `make genproto`, will take a look at why this happened.

Reply:
Thanks for catching this, it was a problem with the sed patch not replacing `StringKeyValue` correctly, fixing now.